From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tobias Christiansen <tobyase@serenityos.org>
Date: Fri, 11 Mar 2022 19:01:35 +0100
Subject: [PATCH] Include fcntl find fcntl.h

`fcntl.h` was included as `sys/fcntl.h`, which is not where this lives in Serenity.

Also `sys/select.h` is included here.
---
 src/Common.hh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/Common.hh b/src/Common.hh
index 2482a13..6476f11 100644
--- a/src/Common.hh
+++ b/src/Common.hh
@@ -26,7 +26,8 @@
 #include <netinet/in.h>
 #include <sys/un.h>
 #include <sys/stat.h>
-#include <sys/fcntl.h>
+#include <fcntl.h>
+#include <sys/select.h>
 
 #ifdef ENABLE_NLS
 #include <libintl.h>
